cloud/logger/example/index.ts

```diff
@@ -1,20 +1,37 @@
 
-import {BunyanLogger, ConsoleLogger, Logger} from '../src/index';
+import { BunyanLogger, ConsoleLogger, getLogger, logger, Logger, setLogger} from '../src/index';
```

Comment:
⚠️  [Antipattern] Exposing methods on the top level module - we can wrap them into object. 
:stop_sign: [Antipattern] Exposing both getter and instance.